packages/shared/routes/setup/migrate/views/Migrate.svelte

```diff
@@ -111,7 +117,8 @@
                                 newProfile.set(null)
                             }
                         })
-                        .catch((error) => {
+                        .catch((err) => {
+                            const error = err?.message ? err.message : err?.toString()
```

Comment:
By adding this line the notification message for ledger devices is generic now (Something went wrong) instead of correct error by status that gets resolved in `getLegacyErrorMessage` 

```
showAppNotification({
                                type: 'error',
                                message: locale(getLegacyErrorMessage(error)),
                            })
```
example for canceling transaction by clicking Deny on Ledger device:
Current:
![image](https://github.com/iotaledger/legacy-migration-tool/assets/52735957/88d69e7e-2897-47f1-b391-d43e652a02b9)

How it was before:
![image](https://github.com/iotaledger/legacy-migration-tool/assets/52735957/6eddf3eb-eaff-4776-972a-83f4917375a2)